core/embed/rust/src/micropython/obj.rs

```diff
@@ -385,11 +406,39 @@ impl TryFrom<Obj> for usize {
     }
 }
 
-impl<T> From<Option<T>> for Obj
-where
-    T: Into<Obj>,
-{
-    fn from(val: Option<T>) -> Self {
+impl<const N: usize> TryFrom<Option<Vec<u8, N>>> for Obj {
```

Comment:
Can't we have both `From<Option<T>>` and `TryFrom<Option<T>>` defined in generic way? Why do it for each type separately?